Enterprise-CMCS/macpro-mdct-qmr
# [QMR] New Measure - LSC-CH

### Description
Creation of a new measure for FFY23

### Related ticket(s)
[MDCT-2426](https://qmacbis.atlassian.net/browse/MDCT-2426)

---
### How to test
1. Login
2. Create child core set
3. Scroll to LSC-CH measure
4. Go through field check list provided on ticket https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2426

---
